Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -196,7 +196,9 @@
     Telecrystal: 150 # goob
   categories:
   - UplinkWeaponry
-  saleLimit: 0 # WD EDIT
+  conditions:
```

Indicate funky change.
```suggestion
  conditions: # Funky war ops whitelisting
```